docs/contributing/coding-guidelines/ros-nodes/task-scheduling.md

```diff
@@ -1,5 +1,117 @@
 # Task scheduling
 
-!!! warning
+## Scheduing in autoware
 
-    Under Construction
+The software of autoware is the system of multiple nodes.In ROS2 the executor uses one or more threads of the underlying operating system to invoke the callbacks. The different type of callback has the different priority.  
+In autoware,there are two types about "publish-subscribe":
```

Comment:
```suggestion
Autoware is a system made of multiple ROS2 nodes.
In ROS2 the executor uses one or more threads of the underlying operating system to invoke the callbacks.
Callbacks are units of work like subscription callbacks, timer callbacks, service calls, and received client responses.
The different types of callback has the different priority. Here are the priorities for:
```
Reference documentation: https://docs.ros2.org/latest/api/rclpy/api/execution_and_callbacks.html#module-rclpy.executors

Also made some small fixes like spaces after commas and periods.